scripts/release.sh

```diff
@@ -0,0 +1,281 @@
+read user_input
+
+
+echo
+echo "Please perform the following steps"
+# and every step of this process, the user needs complete control.
+# that means we tell them what we want to do, ask them if they want us to do it, report the results, and gracefully handle exits.
+
+# git commands
+#todo - I don't like any of how this section is handled.
+# 8 commit release to develop
+echo "step 8: commit release to develop"
+# git checkout <branch>
+# git add -A
+# git commit -m <message here>
+# git push --set-upstream origin <branchname>
+
+# 8 merge release to master
+echo "8.1 merge release to master"
+# git merge <branch> <master>
+
+# 8 tag it
+echo "8.2 tag the release"
+# git tag <something>
+
+# 8 merge master back to develop
+echo "8.3 merge master back to develop"
+# git merge <branch> <master>
+
+# 1 Create new release for tag
+echo "Step 1: Create new release for tag"
+# ?
+
+# 1 add highlights of changes
+echo "1.1 add highlights of changes"
+# ?
+
+# 2 push new release to primary cocoapod
+echo "Step 2: push new release to primary Cocoapod"
+#pod trunk push SmartDeviceLink.podspec --allow-warnings
+
+# 3 Push the new release to the secondary cocoapod using command line:
+echo "Step 3: Push the new release to the secondary cocoapod using command line:"
+#pod trunk push SmartDeviceLink-iOS.podspec --allow-warnings.
+read user_input
+
+# 4 Add a binary xcframework archive for manual installation with the following commands
+# TODO - these commands did not work on last test.
+echo
+echo "Step 4: Add a binary xcframework archive for manual installation with the following commands"
+# i
+echo "4.i"
+xcodebuild archive -project 'SmartDeviceLink-iOS.xcodeproj/' -scheme 'SmartDeviceLink' -configuration Release -destination 'generic/platform=iOS' -archivePath './SmartDeviceLink-Device.xcarchive' SKIP_INSTALL=NO
+
+# ii
+echo "4.ii"
+xcodebuild archive -project 'SmartDeviceLink-iOS.xcodeproj/' -scheme 'SmartDeviceLink' -configuration Release -destination 'generic/platform=iOS Simulator' -archivePath './SmartDeviceLink-Simulator.xcarchive' SKIP_INSTALL=NO
+
+# iii
+echo "4.iii"
+xcodebuild -create-xcframework -framework './SmartDeviceLink-Device.xcarchive/Products/Library/Frameworks/SmartDeviceLink.framework/' -framework './SmartDeviceLink-Simulator.xcarchive/Products/Library/Frameworks/SmartDeviceLink.framework/' -output './SmartDeviceLink.xcframework'
+
+# iv Compress the .xcframework and add the it to the release.
+echo
+echo "4.iv Compress the .xcframework and add the it to the release."
+file="SmartDeviceLink.xcframework"
+zip_file_name="SmartDeviceLink.xcframework.zip"
+if [ ! -f "$zip_file_name" ]; then
+    rm $zip_file_name #kill the old zip if present.  Useful for re-running the script
+fi
+#TODO - we should verify file exists before acting on it.  It's just good practice.  Maybe review and apply through this script.
+zip $zip_file_name $file
+#TODO - remove old files
+# rm $file
+#TODO - add framework to release ??
+# git add <filename>
```

Comment:
```suggestion

file="SmartDeviceLink.xcframework"
zip_file_name="SmartDeviceLink.xcframework.zip"
if [ ! -f "$zip_file_name" ]; then
    rm $zip_file_name #kill the old zip if present.  Useful for re-running the script
fi
#TODO - we should verify file exists before acting on it.  It's just good practice.  Maybe review and apply through this script.
zip $zip_file_name $file
# rm $file
```

Reply:
Not sure what you are looking to change here.  I see that you want to remove the echo at the top and at the bottom.